### DISCUSSION TITLE
React 18 Concurrent mode support

### DISCUSSION THREAD

**Discussion by aaron5670** [mantinedev/mantine#753] [Q&A] [answered]
# React 18 Concurrent mode support

Hi all,

Again, I really love this library! 🥇 

Today I tried to install Mantine on a new Next.js environment with React 18 and Concurrent mode. Only this unfortunately did not work yet [Next.js React 18 docs](https://nextjs.org/docs/advanced-features/react-18#enable-ssr-streaming-alpha).

Are there any plans to support React 18 yet?
Since the release of React 18 could happen at any time.

Thanks!

**Comment by rtivital** [comment]
No, I'm not planning to support React 18 until it is not released

**Comment by rtivital** [comment, marked as answer]
I've tried react@18.0.0-rc.0, everything works as expected, you can try it here – https://github.com/mantinedev/mantine-next-template/tree/react-18

**Reply by aaron5670** [reply]
Hi @rtivital,

Thank you for the example!

Sorry, I wasn't quite clear.
I actually meant Next.js with React 18 and Concurrent features enabled [Next.js React 18 docs](https://nextjs.org/docs/advanced-features/react-18#enable-ssr-streaming-alpha).

Thanks you!

**Reply by rtivital** [reply]
No, not planning to support these features until react 18 is out, Mantine depends on emotion to support these features.
